airflow-core/src/airflow/ui/src/i18n/locales/he/dashboard.json

```
@@ -0,0 +1,38 @@
+{
+  "welcome": "ברוכים הבאים",
+  "stats": {
+    "stats": "סטאטוס",
+    "failedDags": "בכשלון Dags",
+    "runningDags": "בריצה Dags",
+    "activeDags": "פעיל Dags",
+    "queuedDags": "Dags בתור"
+  },
+  "health": {
+    "health": "System-Gesundheit",
+    "metaDatabase": "מטא מסד-נתונים",
+    "scheduler": "מתזמן",
+    "triggerer": "Triggerer",
+    "dagProcessor": "Dag מפרסר",
+    "status": "סטאטוס",
+    "lastHeartbeat": "פעימה אחרונה",
+    "healthy": "תקין",
+    "unhealthy": "לא תקין"
+  },
+  "importErrors": {
+    "searchByFile": "חפש לפי קובץ",
+    "dagImportError_one": "שגיאות בטעינת Dag",
+    "dagImportError_other": "שגיאות בטענית Dags",
+    "timestamp": "חותמת זמן"
+  },
+  "poolSlots": "Pool סטאטוס",
+  "managePools": "Pools ניהול",
+  "history": "היסטוריה",
+  "sortBy": {
+    "newestFirst": "החדש ביותר",
+    "oldestFirst": "הישן ביותר"
```

```suggestion
    "newestFirst": "חדש קודם",
    "oldestFirst": "ישן קודם"
```
Or equivalent with pluralization / article